config/mimes.php

```diff
@@ -0,0 +1,6 @@
+<?php
+
+return [
+    'doc'  => array('application/msword', 'application/vnd.ms-office'),
+    'docx' => array('application/vnd.openxmlformats-officedocument.wordprocessingml.document', 'application/zip'),
```

Comment:
application/zip is not a docx file
Also add the valid mimetypes for the other file extensions

Reply:
deleted the file.